lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -196,16 +191,13 @@ def collapse_namespace(namespace, const)
         #   const_name(s(:const, s(:const, nil, :M), :C)) # => [:M, :C]
         #   const_name(s(:const, s(:cbase), :C))          # => [nil, :C]
         def const_name(node)
-          # rubocop:disable InternalAffairs/NodeDestructuring
-          namespace, name = *node
-          # rubocop:enable InternalAffairs/NodeDestructuring
+          namespace, name = *node # rubocop:disable InternalAffairs/NodeDestructuring
           if !namespace
             [name]
           elsif namespace.const_type?
             [*const_name(namespace), name]
           elsif %i[lvar cbase send].include?(namespace.type)
             [nil, name]
-
```

Comment:
Feel free to squash these blank-line removals into the previous commit. I have no idea why I added them there 😬